index.html

```diff
@@ -26,24 +24,259 @@ <h1>Hey there. I'm Richard</h1>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
+            <button class="container-btn" type="button" name="button">See Project</button>
+      </section>
+      <section class="container">
+            <div class="container-image">
+                <img class="item-image" src="images/1234.png" alt="">
+            </div>
+            <div class="container-title">
+                <p>Multi-Post Stories <br> Gain+Glory</p>
+            </div>
+            <div class="container-projects">
+                <div class="container-projects-element">
+                   <a href="#">Ruby on rails</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">CSS</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">Javascript</a>
+                </div>
+                <div class="container-projects-element">
+                   <a href="#">html</a>
+                </div>
+            </div>
```

Comment:
- [ ] The list of languages has a `ul` tag to group them, and `li` per icon. Please change it. Apply it to the frameworks and skill section.